system-test/test-install.ts

```diff
@@ -124,6 +139,22 @@ new LoggingWinston({
     description: 'imports the module with a prefix and labels specified',
     dependencies: ['winston@2'],
     devDependencies: []
+  },
+  {
+    code:
+        `const LoggingWinston = require('@google-cloud/logging-winston').LoggingWinston;
+new LoggingWinston({
+  prefix: 'some-prefix',
+  labels: {
+    env: 'local',
+    name: 'some-name',
+    version: 'some-version'
+  }
+});`,
+    description:
+        'winston 3: imports the module with a prefix and labels specified',
```

Comment:
Nit -- the description here is just a little different (space before `3`) than the previous added case.